app/src/main/java/org/onionshare/android/files/FileManager.kt

```diff
@@ -159,18 +143,16 @@ class FileManager @Inject constructor(
                 }
             }
         } catch (e: CancellationException) {
+            LOG.info("Got cancelled, deleting zip file...")
             zipFile.delete()
             throw e
         }
         currentCoroutineContext().ensureActive()
-        _state.value = Zipping(files, zipFile, 100)
+        _zipState.value = ZipState(zipFile, 100)
         zipFile.deleteOnExit()
```

Comment:
Separate issue, but does this work on Android?

Reply:
It isn't guaranteed to, hence: https://github.com/onionshare/onionshare-android/issues/10